### PR TITLE
Added tooltip for combined compound score

### LINKED ISSUE #4560: Add a tooltip on variant's page, compounds table to explain Combined score
I got the question via email:

image

Explain that it's the sum of variants's score and compound variant's score

Don't forget to add the toolptip to all pages that have that table

### PR DESCRIPTION
**Pull request by TereseBo**
This PR adds a tooltip for the "combined score" header in the compounds table. For technical reasons the tooltip is conditionally rendered so it's not included when the table is itself displayed in a popover.

This PR closes #4560

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [x] tests executed by TB

### CODE REVIEW

**Comment by TereseBo** [comment]
On main branch, the table is visible both on the variant page and as popover triggered by the compound-bade in the variants list:
image

image

On this branch, the popover table no longer includes un-clickable links:
image
 
In the variants page, links are rendered as before and a question mark which displays a tooltip on hover has been added:
image

**Review by northwestwitch — COMMENTED** [review]
Perhaps I'm missing something but it doesn't work?

File: scout/server/blueprints/variants/templates/variants/utils.html [hunk 1]
```diff
@@ -131,6 +136,8 @@
           <td>
               {% if compound.not_loaded %}
                 {{ compound.display_name }} <small>(not loaded)</small>
+              {% elif is_popover %}
```
Comment (northwestwitch) [inline scout/server/blueprints/variants/templates/variants/utils.html:139]: 💯

**Review by northwestwitch — APPROVED** [review]
Works great, thanks! 🥇

**Pull request merged**